recipes/fish-shell/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "fish-shell" %}
```

Comment:
1. I thought it might be good to make it directly clear that it is a shell, since installing shells probably is rather uncommon for conda users (except for xonsh of course 😉, whose name is unique unlike "fish")
2. `pip search fish` delivers many results, including one with this exact name: https://pypi.org/project/fish/.

I know these two points are not very strong and there is also a good reason to change the name to just "fish": It may be less confusing to users if the name of the package exactly matches the name of the executable. So I am ok with changing it to "fish".